modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -1,5 +1,6 @@
-import AposInputMixin from 'Modules/@apostrophecms/schema/mixins/AposInputMixin';
 import { klona } from 'klona';
```

Comment:
Do we still need klona?

Reply:
yes:

![image](https://github.com/user-attachments/assets/22d0e0b7-e082-4f76-b4d6-1b72e08566ed)